applications/integrationtests/mechanics/Feti.cpp

```diff
@@ -23,6 +23,8 @@
 #include "mechanics/sections/SectionPlane.h"
 #include "mechanics/integrationtypes/IntegrationTypeEnum.h"
 
+using std::cout;
+using std::endl;
 using NuTo::Constitutive::ePhaseFieldEnergyDecomposition;
```

Comment:
maybe use `using namespace NuTo::Constitutive` in a scope. Maybe only in `AssignMaterial`?